README.md

```diff
@@ -145,4 +145,41 @@ The above script builds the necessary docker images and starts the SpringBoot se
 The [ITK Testbench](https://digital.nhs.uk/services/interoperability-toolkit/developer-resources/itk-test-centre/itk-testbench) provided by NHS Digital has an essential tool for ITK interface development and conformance.  This tool provides resources that will allow you to adequately test the 111 adaptor within your own environment.
 
 ## Test Pack
-Our testing pack as well our test harness can be found inside the test-suite directory [Test-Suite](./test-suite)
\ No newline at end of file
+Our testing pack as well our test harness can be found inside the test-suite directory [Test-Suite](./test-suite)
+
+## Running Test Harness
```

Comment:
This is how we (NHS England) would run the test harness, but unlikely to be how an NME would run the test harness. They would likely be running it against the hosted version of the adaptor.

Is this not a duplicate of the instructions within the test-suite/README.md file?